frontend/src/pages/recruitment/Final.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import DateFnsUtils from "@date-io/date-fns";
+import MaterialTable from "@material-table/core";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import { Helmet } from "react-helmet";
+import { Visibility } from "@material-ui/icons";
+import { useDispatch } from "react-redux";
+import { TableIcons } from "../../components/Icons";
+import LoadingContainer from "../../components/LoadingContainer";
+import PageContainer from "../../components/PageContainer";
+import { getApplications } from "../../services/applications";
+import { withAuthorization } from "../../components/HOC";
+import { getApplicationReviews } from "../../services/reviews";
+
+
+
+const Applications = () => {
+  const [state, setState] = React.useState({
+    // Initial backend data
+    loading: true,
+    applications: [],
+    // User input
+    start_date: new Date(new Date().getFullYear(), new Date().getMonth() - 9, new Date().getDate()),
+    end_date: new Date(),
+  });
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      const { ok, data } = await getApplications(state.start_date, state.end_date, false, "Final");
+      if (ok) {
+        const applications = data.applications.map((app) => ({
+          ...app,
+
+          role: app.role.name,
+        }));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+          applications,
+        }));
+      } else {
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+        }));
+      }
+    };
+    if (state.loading) {
+      loadData();
+    }
+    const loadReviews = async () => {
+      const reviews = state.applications.map((x) => x);
+      console.log(reviews.length);
+      let i;
+      for (i = 0; i < reviews.length; i++) {
+        const curr = i;
+        const review_list = await getApplicationReviews(reviews[curr]._id);
+        reviews[curr][
+          "resume_rating"
+        ] = `${review_list.data.reviews[0].reviewer.name}: ${review_list.data.reviews[0].rating}/5`;
+        reviews[curr][
+          "phone_rating"
+        ] = `${review_list.data.reviews[1].reviewer.name}: ${review_list.data.reviews[0].rating}/5`;
```

Comment:
the review_list.data.reviews[0] should be review_list.data.reviews[1], and same for the one below. It should be review_list.data.reviews[2]

Reply:
I'm just curious, do the reviews come in order? Like do we know that the first review is resume, next review is phone, etc? What sorts it like this?